test/methods/template.js

```diff
@@ -22,6 +22,16 @@ test('$ can use newlines and tab indentations', testScriptStdout, () => $`echo.j
 	bar`, 'foo\nbar');
 test('$ can use newlines and space indentations', testScriptStdout, () => $`echo.js foo
   bar`, 'foo\nbar');
+test('$ can use escaped newlines and space indentations', testScriptStdout, () => $`echo.js foo\
+  bar`, 'foo\nbar');
+test('$ can use escaped newlines and inline tab indentations', testScriptStdout, () => $`echo.js foo\
+	bar`, 'foo\nbar');
+test('$ can use escaped newlines and character escaped tab indentations', testScriptStdout, () => $`echo.js foo\
+\tbar`, 'foo\tbar');
+test('$ can use escaped newlines and character escaped newlines', testScriptStdout, () => $`echo.js foo\
+\n\nbar`, 'foo\n\nbar');
+test('$ can use escaped newlines without indentation', testScriptStdout, () => $`echo.js foo\
+bar`, 'foobar');
```

Comment:
This test case already exists below with test name `$ handles escaped newlines`.

Reply:
Okay, I removed mine.